pkgs/development/ocaml-modules/charInfo_width/default.nix

```diff
@@ -1,22 +0,0 @@
-{ lib, fetchzip, buildDunePackage, camomile, result }:
```

Comment:
`charInfo_width` has a 2.0 release that should build https://github.com/kandu/charInfo_width/releases/tag/2.0.0